doc/source/user_guide/data_types.rst

```diff
@@ -124,6 +124,45 @@ may write::
    >>> plt.imshow(out)
 
 
+Working with OpenCV
+===================
+
+It is possible that you may need to use an image created using ``skimage`` with
+OpenCV or vice versa. OpenCV image data can be accessed (without copying) in
+NumPy (and, thus, in scikit-image).
+OpenCV uses BGR (instead of scikit-image's RGB) for color images, and its
+dtype is uint8 by default (See `Image data types and what they mean`_).
+
+Converting BGR to RGB or vice versa
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The color images in ``skimage`` and OpenCV have 3 dimensions: width, height and
+color. RGB stands for Red Green Blue. BGR uses the same color space, except the
```

Comment:
Although it might clutter things, I would add a note here that in scikit-image we usually refer to `rows` and `columns` instead of `width` and `height` (which implies a different ordering of the dimensions). See [here](http://scikit-image.org/docs/dev/user_guide/numpy_images.html#coordinate-conventions).